integration/command.go

```diff
@@ -113,9 +113,9 @@ func DeployInspektorGadget(image, imagePullPolicy string) *Command {
 
```

Comment:
IIUC [the documentation](https://github.com/kubernetes-sigs/security-profiles-operator/blob/v0.6.0/installation-usage.md#label-namespaces-for-binding-and-recording), the webhook is only necessary for `ProfileBindings` and `ProfileRecordings` operations. However, given that we are using none of those operations, we don't need to apply any special label to our namespaces. In this way, the webhook will not infer with any operation in our namespaces (and sometimes makes the pod creation fail). Am I right?

Reply:
Or maybe we leave everything as it is since we are doing [`bestEffortResourceMgmt`](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/integration/command.go#L160) already.

Reply:
I suggest making one change at a time. Let's keep it running with this new release and see how it works.